examples/v1beta1/metrics-collector/custom-metrics-collector.yaml

```diff
@@ -1,10 +1,10 @@
 # TODO (andreyvelich) This metrics collector image (kubeflowkatib/custom-metrics-collector) doesn't work in v1beta1.
 # It is currently using api.v1.alpha3.Manager instead of api.v1.beta1.Manager to report metrics.
```

Comment:
Does this issue resolved ?

Reply:
Not yet, this is the tracking issue: https://github.com/kubeflow/katib/issues/1263.